datafusion-cli/Cargo.lock

```diff
@@ -1,2531 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
I think the idea behind this file was to allow datafusion-cli to be built by itself. https://github.com/apache/arrow-datafusion/pull/2112 by @happysalada 

However, I agree it is getting annoying in my local checkout as well (seems to change regularly)

Reply:
Oh, I did not mean to delete this file. I think we want to keep lock files for binaries.